### PR TITLE
Table fixes

### PR DESCRIPTION
**Pull request by avorobyov-semrush**

### CODE REVIEW

File: semcore/table/src/style/table.shadow.css [hunk 1]
```diff
@@ -114,11 +114,19 @@ SRow {
...
   background-color: #fff;
 
-  &:hover SCellRow {
+  &:hover > SCellRow {
     background-color: color-mod(var(--stone) blend(#fff 90%));
   }
 }
 
+SRow[theme='default']:hover,
+SRow[theme]:hover SCellRow[theme='default'],
```
Comment (s-kobets) [inline semcore/table/src/style/table.shadow.css:123]: maybe `SRow[theme]:hover SCellRow[theme='default']`

Comment (avorobyov-semrush) [reply semcore/table/src/style/table.shadow.css:123]: image
I'll fix it

**Comment by s-kobets** [comment]
Those changes broken hover, if cell will be wrapped in component

**Comment by avorobyov-semrush** [comment]
what type of wrapper?

**Comment by s-kobets** [comment]
We updated package with this change https://github.com/semrush/intergalactic/commit/1a5a03f5302d58620e8ed20248b9c7d9a69787c8

**Pull request closed without merging**